### PR TITLE
javascript modules: improve text

### PR DESCRIPTION
**Pull request by borracciaBlu**
### Description
I was reading the module article and i thought i could help.

### Motivation
Gratitude i guess. I always use mdn!

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
Hi! This PR merely adds a few paragraph breaks, which IMO is  unnecessary. MDN style favors continuous prose rather than chopped-up text—the latter is great for skimming but less so for in-depth explanations.

Another reason is because I'm already preparing a modules guide rewrite locally, where this section will be entirely re-written. As such, let's keep this as-is for now. Thank you for your contribution still; hope you'll make more in the future.

**Comment by borracciaBlu** [comment]
Hi @Josh-Cena ,

thanks for the review and the explanation.

**PR Motivation**
Please, let me give you more context for the PR as i feel i've been shallow in the motivation section; hopefully it will help in the aim of a rewrite.

The changes where done to increase *content accessibility*. 
This is done using short sentences, minimise punctuation,  use active voice and remove unnecessary words or phrases.

The two sentences changed from grade 14 and grade 15 to grade 9 and grade 11 - measured with [Hemingway App](https://hemingwayapp.com/).

Consider that some [institutional websites](https://www.education.vic.gov.au/publish/Pages/contentwrite.aspx) recommend grade 8 or 9 - I mention this is  just to show that less is better as easier to access.

So, despite [CONTRIBUTING.md](https://github.com/mdn/content/blob/main/CONTRIBUTING.md) not explicitly referring to *content accessibility* nor *Hemingway App* i felt it was a good step to make the content more accessible to people with different cognitive abilities as easier to understand (from grade 15 to grade 11).

**Future PR**
Considering that *MDN style favors continuous prose*, do you think that future PRs in the same spirit (sentence editing to increase *content accessibility*) may be considered unnecessary?


Thanks again.
Regards

**Comment by Josh-Cena** [comment]
Readability improvements would be judged on a per-case basis, but I would advise against using a standard metric to measure it. Sometimes a sentence is longer, but it's so idiomatic that many people would find it more natural to read at a glance. Sometimes it introduces weird rhythms that make people pause for a second.

There are many PRs that improve readability that get merged, but eventually it's down to the reviewer's discretion of what's more readable and what fits the MDN style better. There's no general rule for this.

File: files/en-us/web/javascript/guide/modules/index.md [hunk 1]
```diff
@@ -13,13 +13,19 @@ This guide gives you all you need to get started with JavaScript module syntax.
...
+JavaScript programs started off pretty small — most of its usage in the early days was to do isolated scripting tasks, providing a bit of interactivity to your web pages where needed.
 
-It has therefore made sense in recent years to start thinking about providing mechanisms for splitting JavaScript programs up into separate modules that can be imported when needed. Node.js has had this ability for a long time, and there are a number of JavaScript libraries and frameworks that enable module usage (for example, other [CommonJS](https://en.wikipedia.org/wiki/CommonJS) and [AMD](https://github.com/amdjs/amdjs-api/blob/master/AMD.md)-based module systems like [RequireJS](https://requirejs.org/), and more recently [Webpack](https://webpack.js.org/) and [Babel](https://babeljs.io/)).
+Large scripts were generally not needed.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/modules/index.md:18]: For example, this does not make sense to be on its own paragraph. It's not something the reader would care outside the context of the paragraph it used to be in.

File: files/en-us/web/javascript/guide/modules/index.md [hunk 2]
```diff
@@ -13,13 +13,19 @@ This guide gives you all you need to get started with JavaScript module syntax.
...
+
+Fast forward a few years and we now have complete applications being run in browsers with a lot of JavaScript, as well as JavaScript being used in other contexts — [Node.js](/en-US/docs/Glossary/Node.js), for example.
+
+It made sense start thinking about splitting JavaScript programs up into separate modules that can be imported when needed.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/modules/index.md:22]: This is less grammatical. "Make sense to start thinking" is better. I also don't think removing "therefore" is particularly valuable.

File: files/en-us/web/javascript/guide/modules/index.md [hunk 3]
```diff
@@ -13,13 +13,19 @@ This guide gives you all you need to get started with JavaScript module syntax.
...
+
+Node.js had this ability for a long time, and a number of JavaScript libraries and frameworks for module usage is already there — for example, [CommonJS](https://en.wikipedia.org/wiki/CommonJS), [AMD](https://github.com/amdjs/amdjs-api/blob/master/AMD.md)-based module systems like [RequireJS](https://requirejs.org/), and more recently [Webpack](https://webpack.js.org/) and [Babel](https://babeljs.io/).
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/modules/index.md:24]: It should be "has had this ability", because they still do!

**Review by Josh-Cena — COMMENTED** [review]
These are a few examples of why I think the change does not prove an immediate quality improvement.

**Pull request closed without merging**